packages/opentelemetry-node/package.json

```diff
@@ -64,6 +64,7 @@
   },
   "types": "types/index.d.ts",
   "dependencies": {
+    "@opentelemetry/core": "1.27.0",
```

Comment:
Perhaps a caret dep, like the others?  Also, FWIW I notice that the ".../resources" dep below is `^1.26.0`. I don't think that matters. It depends on the package-lock.

Reply:
I was hesitating but I'm going to add it